configs/nvme-storage-configurator/base/daemonset.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nvme-storage-configurator
+  labels:
+    app: nvme-storage-configurator
+spec:
+  selector:
+    matchLabels:
+      app: nvme-storage-configurator
+  template:
+    metadata:
+      labels:
+        app: nvme-storage-configurator
+    spec:
+      nodeSelector:
+        konflux-ci.dev/storage: "nvme"
```

Comment:
so we also need to add a node pool for those machines right?

Reply:
Yes, we need to add this label for the new node pool with nvme disks.